net-misc/s6-networking/s6-networking-2.5.1.0.ebuild

```diff
@@ -17,16 +17,16 @@ IUSE="ssl static static-libs"
 REQUIRED_USE="static? ( static-libs )
 	ssl? ( !static !static-libs )"
 
-RDEPEND=">=dev-lang/execline-2.8.1.0:=[static-libs?]
-	>=dev-libs/skalibs-2.11.0.0:=[static-libs?]
-	>=sys-apps/s6-2.11.0.0:=[execline,static-libs?]
+RDEPEND=">=dev-lang/execline-2.8.1.0:=[static-libs(-)?]
```

Comment:
git mv (for both static-libs bit and dep bound, for latter in particular, think of e.g. mixed stable/~arch etc.)

Reply:
Just in case if I understand correctly, I should have done revbump for all packages where I have changed 
```diff
-       >=dev-libs/skalibs-2.11.0.0:=[static-libs?]
+       =dev-libs/skalibs-2.11*:=[static-libs?]
```
for those who use `dev-libs/skalibs ~arch` but stable reverse dependencies, right? Wouldn't the slot operator do the job in this case? And I should have done revbump for all packages where I have changed `static-libs` to `static-libs(-)` as well.

Reply:
I suppose in that case, := might indeed save us. I'd still revbump to make things easier though. It's acceptable not to in such a case though.

Plus I don't know (not checked) if subslot changed b/t the bad-and-good version. I'm assuming it did.

Reply:
done